api/management/README.md

```diff
@@ -0,0 +1,125 @@
+# Services management
+
+The API described here are intended to assist with the operation and management
+of a Veraison services deployment. They are not meant to be utilized by the
+services' users.
+
+## Policy management
+
+Policies allow augmenting attestation scheme generated results on
+per-deployment basis, by specifying additional rules to be applied to these
+results. This may result in the change of the trust vector values (and,
+correspondingly, the overall result status), and/or addition of supplementary
+claims under the `policy-added-claims` EAR extension.
+
+A policy is defined by a text document describing the rules to be applied by
+the policy engine. Currently, Veraison only support the OPA policy engine.
+Correspondingly, policies must be written in Rego language used by OPA. See
+[OPA policy README] in services repo for details.
+
+Policies are automatically retrieved, based on the attestation scheme and
```

Comment:
Is Line 20 necessary in the API document. This line and part of Line 21 is explaining the internals of the Veraison operation.

Reply:
How the policies are accessed for use is relevant to how they are managed. This does not go into any details however the fact that 
- policies are retrieved automatically (i.e. if a policy exists, it _will_ be used)
- they are keyed on scheme and tenant id
- only one version is "active" at a time

seem relevant to the users of the management interface.